### ISSUE TITLE
RFC-2 Network Realignment: NO free tokens

### ISSUE THREAD

**Issue by Sunsayzxc** [freight-chain/rfc#6]
**Main idea**

There should be NO airdrops, compensations, gifts or other form of giving away tokens, existing ones or the new ones for free (or for the price significantly different (more than 20%) from open market exchange rate), provided by the company, outside prior open market or OTC deal of those tokens purchase by the company. 
After such buyback, there should be a community vote for each particular case of such airdrop, compensation, marketing giveaway or any other form of giving away tokens by the company for free.

**Argumentation**

Most people never value anything they receive for free or with a huge discount. Especially, when we talk about tokens. 
Imagine the following case:
Person decided that he dont want to hold tokens of particular project and exchanged those tokens for equity instead. He already decided that he doesn't need those tokens. He doesnt believe those tokens are valuable. He believes in equity. 
Shortly after, initial terms change. And he recieves those worthless, in his opinion, tokens back. And those tokens become tradeable. What do you think gonna happen? Is he gonna hold a worthless asset or just dump it right away at any price? That depends on his personal thoughts. But there is a huge chance he either gonna dump those right away, or hold a bit and dump those shortly after.

As you see from this example, my idea is that at least some people won't hold this free airdrop tokens. And just dump tokens at any price. Providing additional sell pressure. Amount of such a pressure depends on how many tokens were exchanged for equity. 

On my personal calculations, there where tens of millions tokens exchanged for equity during swap process. Both ARY and BOL. Considering current circulating supply of 88mil, which equals initial circulating supply if the main proposal pass, we can't allow such a huge dilution of our tokens holdings. By huge, I mean 100%+. You hold 1% of whole supply? If we decide to make this equity compensation, you'll hold just 0.5% of total supply. 10k now equals 5k and so on.

People who exchanged for equity already made their choice and there final decision about what they value and trust. Some of us exchanged all of our tokens for equity. Some of us just changed partly. But no matter what, we shouldn't support those tens of millions free tokens get into the market. Otherwise, we risk to have an extremely huge dilution of our tokens holdings and, as a result, a huge drop in price.

**Conclusion**

Giving away free tokens in any form will only result in damaging tokens price, and, as a result, provide a significant negative impact on long term project goals and project's success. On the other hand, that will give only a minor bonus to those holding equity.

**Comment by tomphilly** [comment]
I get your point but there’s no free token. It’s being accounted for the dilution of the token.

**Comment by Sunsayzxc** [comment]
> I get your point but there’s no free token. It’s being accounted for the dilution of the token.

I don't get your point actually) Dilution - is exactly the thing of giving away free tokens. For example. We had 88mil supply. And will have the same, if no bonuses, as an initial supply of new token. But if bonuses pass, a person who don't want to hold tokens and just want equity will receive lets say 15mil tokens, as a payback for what he paid for his equity. And its just 1 person. Could be many. 
But he doesn't want those tokens. And there is a high chance he just dumps those. And you can't calcualte such an effect of this thing if you don't have actual numbers of how many tokens were swapped for equity. But according to Sam himself its tens of millions. 
And even if people who dont want equity gonna have x3 from the amount they have, its still wont ever be enough to compensate those tens of millions of free tokens flow into the market. 
Onve again. Those gonna be flow of new tokens. Which arent supposed to exist in the first place. From people, who dont want to hold those tokens, who believe in equity. Effect of giving away such a huge tokens amount into the hands of people who dont value those could be devastating for tokens price.

**Comment by Sunsayzxc** [comment]
It's like you have 2mil tokens for your 1mil as a bonus. And a person with 0 tokens, but huge amount of equity get 10mil tokens. You value your tokens and have a 200% bonus. Person with equity doesnt have tokens and doesnt value those. Only values equity. And he now have +infinity%, cause he had zero before. And can just dump all of those for a quick buck he didnt think he'll ever have.

**Comment by tomphilly** [comment]
We cannot stop anyone from dumping and we should not. Considered that the possibility that the max supply could go pass 600 millions. That’s more than 10 times our max supply right now. Although with increase supply, your percentage of holdings shouldn’t change

**Comment by Sunsayzxc** [comment]
> We cannot stop anyone from dumping and we should not. Considered that the possibility that the max supply could go pass 600 millions. That’s more than 10 times our max supply right now. Although with increase supply, your percentage of holdings shouldn’t change

But it will. Its simple math. IF equity holders receive a refund of their tokens invested in equity. Without any bonuses we will have 88mil initial supply on the new network.
And if give back tokens to those invested in equity we can easily end up with 200mil initial supply. Or even more.

**Comment by FEEDC0DE** [comment]
> **Main idea**
> 
> There should be NO airdrops, compensations, gifts or other form of giving away tokens, existing ones or the new ones for free (or for the price significantly different (more than 20%) from open market exchange rate), provided by the company, outside prior open market or OTC deal of those tokens purchase by the company.
> After such buyback, there should be a community vote for each particular case of such airdrop, compensation, marketing giveaway or any other form of giving away tokens by the company for free.
> 
> **Argumentation**
> 
> Most people never value anything they receive for free or with a huge discount. Especially, when we talk about tokens.
> Imagine the following case:
> Person decided that he dont want to hold tokens of particular project and exchanged those tokens for equity instead. He already decided that he doesn't need those tokens. He doesnt believe those tokens are valuable. He believes in equity.
> Shortly after, initial terms change. And he recieves those worthless, in his opinion, tokens back. And those tokens become tradeable. What do you think gonna happen? Is he gonna hold a worthless asset or just dump it right away at any price? That depends on his personal thoughts. But there is a huge chance he either gonna dump those right away, or hold a bit and dump those shortly after.
> 
> As you see from this example, my idea is that at least some people won't hold this free airdrop tokens. And just dump tokens at any price. Providing additional sell pressure. Amount of such a pressure depends on how many tokens were exchanged for equity.
> 
> On my personal calculations, there where tens of millions tokens exchanged for equity during swap process. Both ARY and BOL. Considering current circulating supply of 88mil, which equals initial circulating supply if the main proposal pass, we can't allow such a huge dilution of our tokens holdings. By huge, I mean 100%+. You hold 1% of whole supply? If we decide to make this equity compensation, you'll hold just 0.5% of total supply. 10k now equals 5k and so on.
> 
> People who exchanged for equity already made their choice and there final decision about what they value and trust. Some of us exchanged all of our tokens for equity. Some of us just changed partly. But no matter what, we shouldn't support those tens of millions free tokens get into the market. Otherwise, we risk to have an extremely huge dilution of our tokens holdings and, as a result, a huge drop in price.
> 
> **Conclusion**
> 
> Giving away free tokens in any form will only result in damaging tokens price, and, as a result, provide a significant negative impact on long term project goals and project's success. On the other hand, that will give only a minor bonus to those holding equity.

I support this proposal/suggestion by Sunsayzxc.

I'm a new buyer (100%), but I would support some reasonable bonus for old holders, but only for the amount they can prove without doubt they hold, without selling, since TGE+1 year (daily traders≠long term holders).

**Comment by Brandonme** [comment]
Agreed on this, i am pretty much against any move that unnecessary adds to token dilution, not only is it ridiculous to even consider giving back tokens to those who sent for equity, but also shady as some kind of back room deal, people split their tokens for nodes or equity in ratio depending on how they saw value in each and at the time no one knew they could get their tokens back. 

Also the problem is node hosting rewards for first year, I don't think 60m token increase thru node staking is correct as it will seem like some high inflation coin, id rather see a part of that supply added to token holders (as i think someone said 35-50%) and in turn increase the node hosting token requirements, to adjust with the increased token supply. So more of the supply is locked up in nodes.

**Comment by sambacha** [comment]
@all-contributors please add @Brandonme  for content

@all-contributors please add @FEEDC0DE for content 

@all-contributors please add @Sunsayzxc  for ideas

@all-contributors please add @tomphilly  for content

**Comment by sambacha** [comment]
@all-contributors please add @FEEDC0DE for content 

@all-contributors please add @Sunsayzxc for ideas

@all-contributors please add @tomphilly for content

**Comment by sambacha** [comment]
@all-contributors please add @Sunsayzxc code, doc

**Comment by sambacha** [comment]
@all-contributors please add @FEEDC0DE for doc

@all-contributors please add @Sunsayzxc for doc

@all-contributors please add @tomphilly for doc

**Comment by sambacha** [comment]
@all-contributors please add @FEEDC0DE for doc

**Comment by sambacha** [comment]
@all-contributors please add @tomphilly for content
